File: 프로그래머스/unrated/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.py
def solution(myString, pat):
    
    cnt = 0
    for i in range(len(myString)):
        s = ''
        for j in range(i, len(myString)):
            s += myString[j]
            
            if s == pat:
                cnt += 1
                s = ''
                break
    
    return cnt

File: 프로그래머스/0/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.java
class Solution {
    public int solution(String myString, String pat) {
        int answer = 0;
        
        for(int i = 0; i < myString.length() - 1; i++) {
            String tmp = myString.charAt(i) + "";
            for(int j = i + 1; j < myString.length(); j++) {
                tmp += myString.charAt(j) + "";
                if (tmp.equals(pat)) {
                    answer++;
                    break;
                }
            }
        }
        
        return answer;
    }
}